concrete/src/Foundation/Runtime/Run/DefaultRunner.php

```diff
@@ -151,6 +158,16 @@ protected function initializeTimezone()
         @date_default_timezone_set($config->get('app.server_timezone'));
     }
 
+    /**
+     * Initialize localization.
```

Comment:
Can you update the documentation for this method to match the other protected methods like this one?